truffleshuffle/build.gradle

```diff
@@ -9,7 +9,7 @@ android {
 
```

Comment:
can you also change the compileSDK version in this file, as well as in the app/build.gradle file

Reply:
I have made the changes to app/build.gradle file and have also updated the compileSDK